portfolio/src/main/webapp/slide.css

```diff
@@ -5,6 +5,10 @@
 
 .pic {
```

Comment:
This should probably have a better name highlighting that the picture is hidden when the style is applied.